arviz/tests/helpers.py

```diff
@@ -165,8 +165,10 @@ def check_multiple_attrs(test_dict, parent):
 
     Args
     ----
-    test_dict: dict
-        Its structure should be `{dataset1_name: [var1, var2], dataset2_name: [var]}`
+    test_dict: dict of {str : list of str}

```

Comment:
What's the motivation for requiring that a variable be absent?

Reply:
In the case of a group, I was thinking in cases such as `log_likelihood` in sample stats, I see more uses to the group case though (e.g. `test_multiple_observed_rv_without_observations` in test_data_pymc where we want to test there is no observed data because there should not be)

Reply:
Possibly change the `test_dict` type annotation to type hint format?  `Dict[str, List[str]]`.

For that matter, you could add type hints, but of course that's not required.

Reply:
I will try to add proper type hinting this evening (following #415) and leave the docstring in pydata style. Some output examples are also in order I think. The `assert error False == ["observed_data", ("sample_stats", "~log_likelihood")]` messages should be useful for figuring what is the problem (in this case, observed data group is expected but not found and log likelihood should not be in sample stats but it is)